scopes/react/react/webpack/webpack.config.preview.dev.ts

```diff
@@ -55,17 +55,20 @@ export default function ({ envId, fileMapPath, distPaths, workDir }: Options): W
         {
           test: /\.js$/,
           enforce: 'pre',
-          include: distPaths,
+          // limit loader to files in the current project,
+          // to skip any files linked from other projects (like Bit itself)
+          include: path.join(workDir, 'node_modules'),
+          // only apply to packages with componentId in their package.json (ie. bit components)
+          descriptionData: { componentId: (value) => !!value },
```

Comment:
This is the source-map-loader rule for the Preview (docs / compositions).
I could check `ComponentID.isValidObject` componentId is a valid component object, but it's not necessary for the `soure-map-loader`